actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -2,7 +2,9 @@
 
 # :markup: markdown
 
-require "rails-html-sanitizer"
+unless defined?(Rails::HTML::Sanitizer)
+  require "rails-html-sanitizer"
+end
```

Comment:
It prevents circular require warnings when executing `actiontext`, `activestorage`, and `actionbox`.

Reply:
If you can reproduce this outside of this context could you send a PR?

Reply:
Sure, I'll do more investigation on that, too.